app/server/src/main/scala/org/bitcoins/server/WalletRoutes.scala

```diff
@@ -862,6 +863,58 @@ case class WalletRoutes(wallet: AnyDLCHDWalletApi)(implicit
           }
       }
 
+    case ServerCommand("exportseed", arr) =>
+      withValidServerCommand(ExportSeed.fromJsArr(arr)) {
+        case ExportSeed(walletNameOpt, passwordOpt) =>
+          complete {
+            val seedPath = kmConf.seedFolder.resolve(
```

Comment:
This pattern is used in a ton of places, can we refactor this to a helper method somewhere?